init.sh

```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env bash
 
-echo "🚢 Build docker images"
-docker compose build
-
 echo "📦 Starting python app"
```

Comment:
Not sure this comment should go here, but I think that `init.sh` should be doing the setup that needs to be done on the workstation given that you have poetry installed. (In addition to setting up docker)